### PR TITLE
Add tests with eager execution

### PR DESCRIPTION
**Pull request by dfalbel**

### CODE REVIEW

**Comment by skeydan** [comment]
great, thanks!

**Comment by skeydan** [comment]
I know this is not directly related but comes to mind ... I think Python 3 is never really used for the tests, right?

**Comment by dfalbel** [comment]
I am not sure. I just added a line so we have that printed in travis.

**Comment by skeydan** [comment]
great!

File: .travis.yml [hunk 1]
```diff
@@ -16,29 +16,44 @@ cache:
...
-      env: TENSORFLOW_VERSION="1.13.1"
-    - name: "2.0-preview"
+    - name: "TensorFlow (Stable)"
+      env: TENSORFLOW_VERSION="default"
+    - name: "TensorFlow Eager (Stable)"
       env:
-        - TENSORFLOW_VERSION="https://files.pythonhosted.org/packages/ba/98/b0b9b83c9eddb9a112e0f928481c78232e1675397ccc717d1dcd92b8544b/tf_nightly_2.0_preview-2.0.0.dev20190319-cp27-cp27mu-manylinux1_x86_64.whl"
+        - TENSORFLOW_VERSION="default"
+        - TENSORFLOW_EAGER="TRUE"
+    - name: "Tensorflow (2.0.0-alpha0)"
```
Comment (terrytangyuan) [inline .travis.yml]: nit: Tensorflow -> TensorFlow

**Comment by dfalbel** [comment]
It was actually not finding tensorflow when we try to set python 3. (https://travis-ci.org/rstudio/tensorflow/jobs/524629123#L2348)

In the others we are using 2.7

File: .travis.yml (hunk 1 above)
Comment (dfalbel) [reply .travis.yml]: Fixed!

**Pull request merged**